metricbeat/module/system/_meta/config.full.yml

```diff
@@ -4,10 +4,10 @@
     - cpu
 
     # System Load stats
-    - load
+    #- load
 
     # Per CPU core stats
-    #- core
+    - core
```

Comment:
I can revert this too. I was thinking that it was only commented out because of it not working on Windows.